tests/features/solution/solution.edit.feature

```diff
@@ -0,0 +1,69 @@
+@api
+Feature: Solution editing.
+  In order to manage solutions
+  As a moderator or solution facilitator
```

Comment:
It's a wrong copy-paste, it should say "solution owner or solution facilitator".